### ISSUE TITLE
Please add namespace support for the vscode extension

### ISSUE THREAD

**Issue by arindam-bandyopadhyay** [solo-io/squash#181]
Squash vscode extension is not working. Please find below the version
* Environment:
```
k8s version - 1.12.7
vscode version - 1.33.1
Squash vscode extension version - 0.5.9
OS - macOS High Sierra
```
* Expected output: 
When I click `Squash - debug pod` the list of all pods should show up in the vscode UI
![image](https://user-images.githubusercontent.com/9819737/56579345-8edccb80-65ed-11e9-8c80-ca0760f01058.png)
* Actual output: 
Instead of getting list of pods from the vscode UI as a list I am getting the json list of the pods in the vscode notification
![image (1)](https://user-images.githubusercontent.com/9819737/56579387-a87e1300-65ed-11e9-8eff-f0796cc5fc02.png)
Please find the vscode logs for more details
![Screen Shot 2019-04-23 at 5 46 54 PM](https://user-images.githubusercontent.com/9819737/56580312-e7ad6380-65ef-11e9-8aea-8e3b5aabcd2d.png)
Please note that squashcli is working fine for me from terminal. Only the vscode extension is not working for me.

**Comment by arindam-bandyopadhyay** [comment]
@mitchdraft I have spent some time in debugging the issue. You are trying to list all the pods in all the namespace using `kubectl get -o json pods --all-namespaces`. My k8s cluster is running inside a VPN and this command takes more than 2 minutes in my environment.  I have more than 1200 pods running in my environment. As a result, the operation is timed out(I guess) and though it can fetch the json, it's printing the json in the vscode notification instead of parsing and showing 1200 pods
Don't you think it would be much nicer if you ask the user first the `k8s namespace` and then list all the pods in that namespace? The squashctl also does that. 
```
$ squashctl
Attaching debugger
? Select a debugger dlv
? Select a namespace to debug  [Use arrows to move, type to filter]
❯ adm-console
  concourse-main
  cplane-prometheus
  default
  hosts
  ns-kcaruso-ip
  ns-user-0f64b2ee50ab53bc
```
I tried changing the extension to ask for a namespace and then listing the pods in that namespace. In that way, it's working fine.  I am going to raise a PR with the namespace support for the vscode extension(in that way vscode extension and squashctl behaves similarly)
